src/rules/require-baseline.js

```diff
@@ -625,6 +628,48 @@ export default {
 					});
 				}
 			},
+
+			Selector(node) {
```

Comment:
This matches all selectors, including those in `@supports`:

```css
@supports selector(:nth-child(1 of .foo)) {
  /* Styles */
}
```

When in `@supports`, we should not warn because they are checking for browser support.

Also related to `@supports`: If someone is checking if a selector is supported then we should not warn on any selectors used inside of that `@supports`. We already do this for property names and values, so we should do the same for selectors.

Reply:
Based on https://github.com/eslint/css/pull/61#issuecomment-2676252875 it doesn't seem like CSSTree parses selectors within `@supports` preludes. So if I understand correctly, while we should be able to tell whether the selector is used somewhere within a `@support` rule, we won't necessarily know if the selector was used in the `@support` condition itself.

For example, if an unrelated property is used in the condition, the non-Baseline feature could still be used in the declaration block:

```css
@supports (color: red) {
	h1:has(+ h2) {
		color: red;
	}
}
```

Is that an acceptable false negative? If so I can update the code and add a note in the docs.

Reply:
Ah I just realized that the astexplorer.net tool I was using to check the AST is stuck on v2.0.1 and the selector support we need was added to v3.0.0. I'll take another look at this.

Reply:
Updated

Reply:
Yup, I'd recommend using https://explorer.eslint.org, which uses the same parser version as this plugin.